x-pack/plugins/lens/public/editor_frame_service/editor_frame/state_helpers.ts

```diff
@@ -79,7 +81,7 @@ export async function persistedStateToExpression(
 
   return buildExpression({
     visualization,
-    visualizationState,
+    visualizationState: { ...(visualizationState as object), title, description },
```

Comment:
I'm not sure I like this approach - it assumes a certain state structure we don't guarantee and it seems like it's breaking the current isolation between frame and individual visualizations to magically set state.